pandas/core/internals.py

```diff
@@ -2859,7 +2855,7 @@ def interpolate(self, method='pad', axis=0, inplace=False, limit=None,
                                           placement=self.mgr_locs)
 
     def fillna(self, value, limit=None, inplace=False, downcast=None,
```

Comment:
needs to be after value

Reply:
I think it would be fun to try to step in here since this was dropped, but could you explain more what you mean by "after value" here?  From what I can see all his additions were immediately after the value argument.